lldb-eval/eval_test.cc

```diff
@@ -2698,14 +2907,18 @@ TEST_F(EvalTest, TestPostfixIncDec) {
   EXPECT_THAT(EvalWithContext("$d-- - 1", vars_), IsEqual("1.5"));
 
   ASSERT_TRUE(CreateContextVariable("$p", "(int*)4"));
-  EXPECT_THAT(EvalWithContext("$p++", vars_), IsEqual("0x0000000000000004"));
+  EXPECT_THAT(EvalWithContext("$p++", vars_),
+              IsEqual(Is32Bit() ? "0x00000004" : "0x0000000000000004"));
   EXPECT_THAT(EvalWithContext("$p++ + 1", vars_),
-              IsEqual("0x000000000000000c"));
-  EXPECT_THAT(EvalWithContext("$p--", vars_), IsEqual("0x000000000000000c"));
+              IsEqual(Is32Bit() ? "0x0000000c" : "0x000000000000000c"));
+  EXPECT_THAT(EvalWithContext("$p--", vars_),
+              IsEqual(Is32Bit() ? "0x0000000c" : "0x000000000000000c"));
   EXPECT_THAT(EvalWithContext("$p-- - 1", vars_),
-              IsEqual("0x0000000000000004"));
+              IsEqual(Is32Bit() ? "0x00000004" : "0x0000000000000004"));
+#endif
 }
 
+#ifndef __EMSCRIPTEN__
 TEST_F(EvalTest, TestDereferencedType) {
```

Comment:
What's wrong with this test? It seems to me this should work for `wasm32` too...

Reply:
It's the unconditional comparison with lldb's output here. wasm32 can't run lldb, so we can't do this comparison.